### PR TITLE
Fixes #694

### LINKED ISSUE #694: 🐞[Bug]:  Social Media in Footer Section
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

social media icons in footer section is too small it needs some improvements
Screenshot 2024-06-25 083619

### Expected behavior

https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/assets/168094736/37538be3-fa58-4cfe-a348-a53995126e64

### Add ScreenShots

_No response_

### What browsers are you seeing the problem on?

Chrome

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by puneet426**
## Description

[Please include a brief description of the changes or features added]

##694

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #694

## Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1388,19 +1388,27 @@ <h4>RAPIDOC Newsletter</h4><br>
...
-                <a href="https://www.twitter.com"><div><img src="twitter.png" style="width: 16px; height: 15px;"></div></a>
-              <a href="https://www.linkedin.com" target="_blank"><div><i class="fa-brands fa-linkedin"></i></div></a>
-               <a href="#" target="_blank"><div><i class="fa-brands fa-instagram"></i></div></a>
-              <a href="https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-" target="_blank"><div><i class="fa-brands fa-github"></i></div></a>
-              <a href="+91 1234567890" title="+91 1234567890" target="_blank"><div><i class="fa-solid fa-phone"></i></div></a>
-              <a href="info@rapidoc.com" target="_blank"><div><i class="fa-solid fa-envelope"></i></div></a>
+              <div class="social-wrapper">
+                <div class="social-links">
+                <div style="margin-left: 300px;font-family: sans-serif;"><i>Follow&nbsp;Us&nbsp;:&nbsp;</i></div>
+                <a href="/- fonts.txt" target="_blank">
```
Comment (varshith257) [inline index.html:1394]: @puneet426 Update links with previous ones used

**Review by varshith257 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (puneet426) [reply index.html:1394]: okk

Comment (puneet426) [reply index.html]: done

**Comment by puneet426** [comment]
please add level GSSOC

File: index.html [hunk 2]
```diff
@@ -1388,19 +1388,27 @@ <h4>RAPIDOC Newsletter</h4><br>
...
+                <a href="https://www.instagram.com" target="_blank">
+                  <i class="fab fa-instagram"></i>
+              </a>
+              <a href="https://www.twitter.com" target="_blank">
+                  <i class="fab fa-twitter"></i>
+              </a>
+              <a href="https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-" target="_blank">
+                  <i class="fab fa-github"></i>
+              </a>
+              <a href="https://www.linkedin.com" target="_blank">
```
Comment (varshith257) [inline index.html:1403]: here too

Comment (puneet426) [reply index.html:1403]: here both links are same as previous

**Review by varshith257 — APPROVED** [review]

**Pull request merged**